### ISSUE TITLE
Vector helm chars failed to install release

### ISSUE THREAD

**Issue by metacoma** [mindwm/mindwm-gitops#40]
```
$ kubectl describe release.helm.crossplane.io/team-a-vector
....
Events:
  Type     Reason                        Age   From                                Message
  ----     ------                        ----  ----                                -------
  Warning  CannotCreateExternalResource  2m2s  managed/release.helm.crossplane.io  failed to install release: 1 error occurred:
           * Service "team-a-vector" is invalid: spec.ports: Required value
```

**Comment by metacoma** [comment]
https://github.com/vectordotdev/helm-charts/issues/205

**Comment by metacoma** [comment]
fixed in 8b5993a8
```
$ kubectl get managed
NAME                                       CHART              VERSION   SYNCED   READY   STATE      REVISION   DESCRIPTION        AGE
release.helm.crossplane.io/team-a-vector   vector             0.33.0    True     True    deployed   1          Install complete   6m12s
```
